### PR TITLE
Osd 8517 part 1 -- shellcheck

### PR DESCRIPTION
**Pull request by qhua948**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -18,14 +18,18 @@ isclean:
...
 	$(CONTAINER_ENGINE) build -t $(IMAGE_URI_VERSION) .
 	$(CONTAINER_ENGINE) tag $(IMAGE_URI_VERSION) $(IMAGE_URI_LATEST)
 
 .PHONY: validation
 validation:
 	./hack/schema_validation.sh
 
+.PHONY: shellcheck
+shellcheck:
+	$(CONTAINER_ENGINE) run -v $(shell pwd):/app --entrypoint=/bin/sh koalaman/shellcheck-alpine -c "cd /app/scripts && find . -name '*.sh' -print0 | xargs -0 -n1 shellcheck -e SC2154 "
```
Comment (georgettica) [inline Makefile:31]: You can use the -w/--workdir flag in docker to change the directory.

It's a tiny nit

**Review by georgettica — APPROVED** [review]
Looks good overall, and you can skip this review aswell

**Comment by georgettica** [comment]
Also if you can squash the commits if there is no special meaning for them that would be helpful

**Comment by georgettica** [comment]
/hold
To hold the tests that are failing

**Comment by qhua948** [comment]
```
12:43:45 /usr/bin/docker run -v /srv/volume/jenkins/workspace/openshift-managed-scripts-gh-pr-check:/app --entrypoint=/bin/sh koalaman/shellcheck-alpine -c "cd /app/scripts && find . -name '*.sh' -print0 | xargs -0 -n1 shellcheck -e SC2154 "
12:43:45 Unable to find image 'koalaman/shellcheck-alpine:latest' locally
12:43:45 /usr/bin/docker: Error response from daemon: Get https://registry-1.docker.io/v2/: dial tcp 127.0.0.0:443: connect: network is unreachable.
12:43:45 See '/usr/bin/docker run --help'.
```

**Comment by qhua948** [comment]
image
Looks like we are not allowed to use docker hub images?

**Comment by qhua948** [comment]
Should be good to go now

**Comment by qhua948** [comment]
part II: https://github.com/openshift/managed-scripts/pull/22

**Pull request merged**